pkg/net/connection.go

```diff
@@ -9,4 +9,6 @@ import (
 type Connection struct {
 	Conn          net.Conn
 	RemotePeerKey *crypto.Key
+	IsIncoming    bool
+	IsOutgoing    bool
```

Comment:
One of these should be enough. If it's not incoming it's "probably" outgoing.

Reply:
done